src/lib/pubkey/kyber/kyber_common/kyber_algos.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * Crystals Kyber Internal Algorithms
+ * Based on the public domain reference implementation by the
+ * designers (https://github.com/pq-crystals/kyber)
+ *
+ * Further changes
+ * (C) 2021-2024 Jack Lloyd
+ * (C) 2021-2022 Manuel Glaser and Michael Boric, Rohde & Schwarz Cybersecurity
+ * (C) 2021-2022 René Meusel and Hannes Rantzsch, neXenio GmbH
+ * (C) 2024 René Meusel, Rohde & Schwarz Cybersecurity
+ *
+ * Botan is released under the Simplified BSD License (see license.txt)
+ */
+
+#ifndef BOTAN_KYBER_ALGOS_H_
+#define BOTAN_KYBER_ALGOS_H_
+
+#include <botan/xof.h>
+#include <botan/internal/fmt.h>
+#include <botan/internal/kyber_symmetric_primitives.h>
+#include <botan/internal/kyber_types.h>
+#include <botan/internal/loadstor.h>
+
+namespace Botan {
+
+void kyber_encode_polynomial_vector(std::span<uint8_t> out, const KyberPolyVecNTT& p);
```

Comment:
Likewise with Dilithium, bikeshedding for `namespace Kyber_F` or the like

Reply:
Done.